lib/Sema/CSSimplify.cpp

```diff
@@ -15201,7 +15201,16 @@ ConstraintSystem::SolutionKind ConstraintSystem::simplifyFixConstraint(
                locator.endsWith<LocatorPathElt::SubscriptMember>())
                   ? 1
                   : 0;
-
+    // An overload choice that isn't settable is least interesting for diagnosis.
+    if (auto overload = findSelectedOverloadFor(getCalleeLocator(fix->getLocator()))) {
+      if (auto decl = overload->choice.getDeclOrNull()) {
+        if (auto VD = dyn_cast<VarDecl>(decl)) {
+          if (!VD->isSettableInSwift(DC)) {
+            impact += 10;
```

Comment:
I think this might be too dramatic of an increase, we just need this to be slightly higher then default impact, increase it by too much and it would start competing with other solutions just like what you have in tests where `count` is a method and a property.

Reply:
Yep! I had gotten the mistaken impression that the impact was local to the specific choice. Changed to just `+= 1`.